core/include/usFrameworkEvent.h

```diff
@@ -195,13 +199,13 @@ class US_Core_EXPORT FrameworkEvent
  * @{
  */
 US_Core_EXPORT std::ostream& operator<<(std::ostream& os, FrameworkEvent::Type eventType);
-US_Core_EXPORT std::ostream& operator<<(std::ostream& os, const FrameworkEvent& event);
+US_Core_EXPORT std::ostream& operator<<(std::ostream& os, const std::exception_ptr ex);
```

Comment:
I don't think this overload is a good idea, we should probably use a helper function for retrieving the exception message instead. Problem is, that this overload is not tied to CppMicroServices specific argument types and hence will be selected whenever someone tries to stream an `exception_ptr` and the FrameworkEvent.h header has been included.